docs/config.md

```diff
@@ -229,7 +229,7 @@ The following settings are available:
 : The amount of time to wait (in milliseconds) when initially establishing a connection before timing out.
 
 `aws.client.endpoint`
-: The AWS S3 API entry point e.g. `https://s3-us-west-1.amazonaws.com`.
+: The AWS S3 API entry point e.g. `https://s3-us-west-1.amazonaws.com`. You must prefix the API entry point with either `http://` or `https://`.
```

Comment:
This can be ambiguous, AWS will not work with HTTP only. Better something like this 👇 

```suggestion
: The AWS S3 API entry point e.g. `https://s3-us-west-1.amazonaws.com` (the endpoint must include the protocol prefix e.g. `https://`).
```